### ISSUE TITLE
Event Creation: Unable to set an event image 

### ISSUE THREAD

**Issue by hpdang** [fossasia/open-event-frontend#3994]
**Describe the bug**
When I created a new event, I tried to upload an event image, then clicked 'publish', the photo disappeared. I have tried several times. 

The image that I uploaded: 4096x1885px, 2.5MB

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Create Event' > 'Basic Details'
2. Click on 'Select Event Image'
3. Click 'Publish'
> Image disappears 

**Screenshots**
After successful upload, it showed 'confirm delete' instead of 'delete' in compared with the select logo option. 
![Screenshot from 2020-02-03 22-31-43](https://user-images.githubusercontent.com/1658742/73692846-6c77f280-46d5-11ea-902c-bc622ff6345e.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: chrome and chromium

**Comment by kushthedude** [comment]
@iamareebjamal This is occurring due to https://github.com/fossasia/open-event-frontend/pull/3980 
```js
[Error] Unhandled Promise Rejection: TypeError: undefined is not an object (evaluating 'delete json.data.relationships['general-statistics']')
	(anonymous function) (open-event-frontend.js:40146)
	promiseReactionJob
```

Why is this needed in serialiser ?
```js
 serialize() {
const json = this._super(...arguments);
delete json.data.relationships['general-statistics'];
delete json.data.relationships['order-statistics'];
 return json;
```

**Comment by iamareebjamal** [comment]
To not serialize the statistics

**Comment by iamareebjamal** [comment]
Simply add a try-catch as it is a purely optional change if they are present. And see if it works

**Comment by kushthedude** [comment]
> Simply add a try-catch as it is a purely optional change if they are present. And see if it works

Same error and loader gets stuck at this error

```js
[Error] TypeError: undefined is not an object (evaluating 'delete json.data.relationships['general-statistics']') — event.js:17
	serialize (open-event-frontend.js:40149)
	superWrapper (vendor.js:54243)
```

**Comment by iamareebjamal** [comment]
Then it is not due to that. If you caught the error, then it should not be thrown

**Comment by kushthedude** [comment]
Also once this error is thrown I again need to restart the server, as in
the case of delete/restore.

**Comment by kushthedude** [comment]
> Then it is not due to that. If you caught the error, then it should not be thrown

Ok, I made these changes. All things are working perfectly. The issue with restore is on the server-side and I will open an issue

**Comment by iamareebjamal** [comment]
That issue was solved by what you are reverting in this PR - https://github.com/fossasia/open-event-frontend/pull/4056

I asked you to wrap it in try catch or if statement

**Comment by kushthedude** [comment]
> That issue was solved by what you are reverting in this PR - #4056
> 
> I asked you to wrap it in try catch or if statement

Try-Catch didn't work, thrown the same error. What should I put in condition for if ?

**Comment by iamareebjamal** [comment]
It's either impossible or you have found a bug in Google Chrome, please file a bug here https://bugs.chromium.org/p/chromium/issues/list

If you want to me check what's wrong, push the PR with try catch
